### PR TITLE
Make the "hpc" module work

### PR DESCRIPTION
**Pull request by dmajda**

### CODE REVIEW

File: data/hpc.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
-  - src/HPCMessages.ycp
-  - src/dialog/HPCDialog.ycp
-  - src/hpc.ycp
+moves:
+  - from: "src/HPC{Control,Messages}.ycp"
+    to:   src/modules
+  - from: "src/{HPCMain,hpc}.ycp"
+    to:   src/clients
+  - from: "src/dialog/HPCDialog.ycp"
+    to:   src/include/hpc
```
Comment (jreidinger) [inline data/hpc.yml:11]: you forgot to move scripts to src/bin

Comment (jreidinger) [inline data/hpc.yml:11]: also desktop file from src/config

Comment (dmajda) [inline data/hpc.yml:11]: Argh, I wasn't aware that I have to look also outside the `src` subtree. Both fixed in #52, thanks for catching that.

**Comment by dmajda** [comment]
Second attempt is in #52, closing.

**Comment by dmajda** [comment]
Senond attempt in #54, closing.

**Pull request closed without merging**